### PR TITLE
HL-170

### PR DESCRIPTION
**Pull request by JasminFarhi**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-170)

- Add 4 cards to vaults page

Screen Shot 2023-01-11 at 3 10 16 pm

### CODE REVIEW

File: src/pages/Vaults.tsx [hunk 1]
```diff
@@ -16,9 +19,53 @@ export const Vaults: React.FC = () => {
...
+          title="24H Deposits"
+          data={
+            totalVaultDeposits &&
+            `$${utils.formatting.formatToFourDecimals(
+              ethers.utils.formatEther(totalVaultDeposits)
+            )}`
+          }
+        />
+        <Card
+          title="24H Withdrawls"
```
Comment (brettvniekerk) [inline src/pages/Vaults.tsx:60]: Withdrawals*

**Review by brettvniekerk — APPROVED** [review]
just fix a spelling

File: src/pages/Vaults.tsx (hunk 1 above)
Comment (OptimusOpus) [reply src/pages/Vaults.tsx:60]: Good catch

Comment (JasminFarhi) [reply src/pages/Vaults.tsx]: whoops, thanks haha

**Review by OptimusOpus — APPROVED** [review]
LGTM

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**